resources/views/applicant/application_index.html.twig

```diff
@@ -11,9 +11,8 @@
 
 {% block body %}
 
-	{% include "common/alert" %}
-
-    {% include "common/goc" %}
+	{% include "common/goc" %}
```

Comment:
This is more of a refactoring note, but if we're making the same change a dozen times (in this case reordering two components), these views should all be extending a template that contains these headers.